metatron/build.gradle

```diff
@@ -4,8 +4,21 @@ version '1.0-SPIKE'
 dependencies {
     compile configurations.dropwizard,
             configurations.common,
+            configurations.verify_saml,
+            project(':proxy-node-shared'),
+            'com.sun.xml.bind:jaxb-core:2.3.0',
+            'com.sun.xml.bind:jaxb-impl:2.3.0',
+            "org.opensaml:opensaml-core:$opensaml_version",
```

Comment:
A lot of these opensaml deps are already specified in a configuration in the `build.gradle` in the root and could be replaced here with `configurations.opensaml`. Suspect some of the others may be defined as well.

Reply:
👍